example/mpi_trapezoidal.jl

```diff
@@ -86,6 +86,8 @@ MPITape.print_mytape()
 tape_merged = MPITape.merge()
 if rank == 0 # Master
     MPITape.print_merged(tape_merged)
+    display(MPITape.plot_merged(tape_merged))
+    readline()
```

Comment:
The plot isn't interactive, right? Might be better to just directly save as png/pdf.